### DISCUSSION TITLE
پشتیبانی از ورژن 1

### DISCUSSION THREAD

**Discussion by mehossein** [evryn/laravel-toman#16] [Q&A] [answered]
با سلام 
یه سوال داشتم 
تو داکیومنت فارسی پکیجتون نوشته بودید وضعیت پکیج ورژنه 1 فعال نیستش 
یعنی چی ؟ 
یعنی دیگه پشتیبانی نمیشه و نمیشه ازش استفاده کرد ؟؟

**Comment by AmirrezaNasiri** [comment, marked as answer]
سلام، وقتتون بخیر

پکیج تو 2 تا ورژن ارائه شده. ورژن 1 که قدیمی هست و 2 که اخیرا منتشر شده.
اگه از ورژن 1 استفاده میکردین، همچنان هم میتونین از ورژن 1 استفاده کنین اما دیگه باگ هاش برطرف نمیشه و آپدیتی ارائه نمیشه براش.
اما اگه بار اوله از این پکیج میخواین استفاده کنین یا تو ورژن قدیمی به مشکل خوردین، بهتره از آخرین ورژن (2) استفاده کنین. تو ورژن 2، نحوه استفاده خیلی راحت تر شده و امکانات خوبی هم برای تست نویسی در نظر گرفته شده.

**Reply by mehossein** [reply]
ممنون
